macOS Demo/ViewController.swift

```diff
@@ -29,12 +29,12 @@ class ViewController: NSViewController {
     updateValueLabel()
     errorLabel.stringValue = ""
   }
-
-  override var representedObject: AnyObject? {
-    didSet {
-    // Update the view, if already loaded.
-    }
-  }
+	
```

Comment:
Those lines have been commented because they were now causing an error and didn't seem to have any utility.